app/components/SongCard/index.js

```diff
@@ -90,44 +87,46 @@ const StyledImage = styled.img`
   margin-bottom: 2em;
 `;
 
+const StyledPlayIcon = styled(PlayCircleTwoTone)`
+  && {
+    font-size: 18;
```

Comment:
Missing unit?

Reply:
by default it takes px right, so was considering this

Reply:
This is smomething,

Reply:
fixed the unit issue